build_tools/ccache/fuse_connection.yaml

```diff
@@ -0,0 +1,22 @@
+logging: 
+  type: syslog 
+  level: LOG_INFO 
+  
+components: 
+  - libfuse 
+  - attr_cache 
+  - azstorage 
+
+libfuse: 
+  path: /mnt/azureblob 
+  allow-other: true
+
+cap_add:
+   - SYS_ADMIN
+devices:
+   - /dev/fuse
+
+azstorage: 
+  account-name: eliasteststorageaccount1
```

Comment:
Reminder to change this account name to something more official before merging to `main`. Could leave a TODO for now while working on the `shared/runner-cluster-migration` branch.